JsonUtils/JsonUtils.psm1

```diff
@@ -190,7 +190,7 @@ function Get-JsonDifferenceRecursion
             }
 
             # check if property exists in FromObject (in PS 5.1 we cant support case sensitive keys https://github.com/PowerShell/PowerShell/issues/3705)
-            if ([bool]($FromObject.PSObject.Properties.Name -match [System.Text.RegularExpressions.Regex]::Escape($ToName)))
+            if ([bool]($FromObject.PSObject.Properties.Name -match "^$([System.Text.RegularExpressions.Regex]::Escape($ToName))$"))
```

Comment:
What about using `if ([bool]($FromObject.PSObject.Properties.Name -contains $ToName))` instead. Any issue you see?

Reply:
That would match "subtitle" with "title"

Reply:
Are you looking at my comment that uses -contains on an array? a la: `"one","two" -contains "one"`

Reply:
What about:
`if ([bool]($FromObject.PSObject.Properties.Name -eq $ToName))`

Reply:
So far, I have a draft that passes unit tests using contains and am inclined to go that route. `... -eq $ToName` is similar, but do you see anything that we're missing if I go with the contains route?

Reply:
Ah, hadn't appreciated it was on an array. That ought to work.

Reply:
Ok, great. Will proceed.